src/jetstream/plugins/cloudfoundryhosting/main.go

```diff
@@ -126,6 +126,10 @@ func (ch *CFHosting) Init() error {
 
 		//Force auth endpoint type to remote (CF UAA)
 		ch.portalProxy.GetConfig().ConsoleConfig.AuthEndpointType = "remote"
+		err := ch.portalProxy.InitStratosAuthService(interfaces.Remote)
```

Comment:
Do we need to do the init here? Its done in main.go.

We override the type to remote - so won't the init in main.go do the right thing?

If so, we can remove InitStratosAuthService from the interface we expose to plugins